src/Mod/Draft/DraftTools.py

```diff
@@ -726,7 +726,7 @@ def Activated(self):
                     import Part
                     w = Part.Wire(edges)
                 except:
-                    msg(translate("draft", "Unable to create a Wire from selected objects\n"),mode="error")
+                    msg(translate("draft", "Unable to create a Wire from selected objects")+"\n",mode="error")
```

Comment:
also, please review this as well. Wasn't sure I did that right

Reply:
this is ok